data/input/court.mainnet.js

```diff
@@ -58,7 +58,7 @@ module.exports = {
   },
   subscriptions: {
     feeToken:                      DAI,                          // fee token for subscriptions is DAI
-    feeAmount:                     bigExp(7500, DAI.decimals),   // 7500 fee tokens per subscription period
+    feeAmount:                     bigExp(1, DAI.decimals),      // 1 fee tokens per subscription period
```

Comment:
The idea of making this not 0 is a small speed bump against spam?

Reply:
TBH the main reason is because we don't handle the case for zero fees :/